### DISCUSSION TITLE
Suggestion for new seam option (mix random an aligned seams)

### DISCUSSION THREAD

**Discussion by make3D-official** [OrcaSlicer/OrcaSlicer#5279] [Ideas]
Hi, 

I'm not sure this is the right place for a suggestion, but here goes nothing. 

I love working with the fuzzy skin feature, but unfortunately I don't feel I have full control over the quality. Many of my prints I have Fuzzy skin on the outside walls and smooth insides. 
In order to achieve the highest print quality, I need random seam on the fuzzy size, and aligned seams on the inside. 

As of today, this is not an option (not even when using a modifier). 
This is a feature I would really love to see in the near future if possible. 

For me the Scarf joints or paint seam does not work as I need it do.

**Comment by vgdh** [comment]
May be some surface selection tool for choosing the seam mode it would be a cool feature!

**Reply by make3D-official** [reply]
This would definitely be helpful as well!

**Comment by foresterr** [comment]
I think allowing modifiers to control seam type could solve this case and a lot of similar ones (not all I guess; for this you'd need to have the option to control seam type in painting mode as well). I have opened a feature request for this (https://github.com/SoftFever/OrcaSlicer/issues/5118)
